ui/revenuecatui/src/main/kotlin/com/revenuecat/purchases/ui/revenuecatui/PaywallFooter.kt

```diff
@@ -0,0 +1,74 @@
+package com.revenuecat.purchases.ui.revenuecatui
+
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.PaddingValues
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.rememberScrollState
+import androidx.compose.foundation.verticalScroll
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import com.revenuecat.purchases.ui.revenuecatui.helpers.isInPreviewMode
+
+internal val footerRoundedBorderHeight = 12.dp
+
+@Composable
+fun PaywallFooter(
+    paywallViewOptions: PaywallViewOptions = PaywallViewOptions.Builder().build(),
+    condensed: Boolean = false,
```

Comment:
This `condensed` parameter right now doesn't affect the UI. That will happen in future PRs